lms/djangoapps/instructor/tests/test_ecommerce.py

```diff
@@ -53,6 +66,24 @@ def test_pass_e_commerce_tab_in_instructor_dashboard(self):
         # Coupons should show up for White Label sites with priced honor modes.
         self.assertIn('Coupon Code List', response.content)
 
+    @mock.patch("openedx.core.djangoapps.site_configuration.helpers.get_value", fake_get_value)
```

Comment:
Instead of mocking, use the [SitesMixin](https://github.com/edx/edx-platform/blob/d1a3af9460e0381aeb69a42632902370eb7bd505/openedx/core/djangoapps/site_configuration/tests/mixins.py#L7).